python/edgar_filing_searcher/api/routes/edgarfiling.py

```diff
@@ -0,0 +1,21 @@
+"""APi for web back-end"""
+from datetime import datetime
+from flask import jsonify, Blueprint, request
+from edgar_filing_searcher.models import EdgarFiling
+
+filing_blueprint = Blueprint('edgarfiling', __name__)
+
+
+@filing_blueprint.route('/edgarfiling/<company_id>')
+def get_filings(company_id):
+    """Route for results for search by company id"""
+    date_format = '%Y-%m-%d'
+    start_date = request.args.get('start_date')

```

Comment:
these should be optional. So you should check to see if they exist first, and then apply the filter if they do.